library/onotes.inc

```diff
@@ -1,76 +0,0 @@
-<?php
-/**
```

Comment:
yep, Matthew brought all the code into the pertinent services class, which is where it belongs. It is another small step in OpenEMR's migration from legacy code to modernized classes.